libraries/AP_Notify/AP_Notify.h

```diff
@@ -31,10 +31,13 @@
 #define RGB_LED_LOW     1
 #define RGB_LED_MEDIUM  2
 #define RGB_LED_HIGH    3
+#define BUZZER_ON       1
+#define BUZZER_OFF      0
 
 class AP_Notify
 {
     friend class RGBLed;    // RGBLed needs access to notify parameters
+    friend class ToneAlarm_PX4;    // ToneAlarm_PX4 needs access to notify parameters
```

Comment:
I think this should be a little bit more generic so it also works on linux.... What about adding a `bool buzzer_enabled() const { return _buzzer_enable:; }` method and avoid the friend class?